### ISSUE TITLE
A11y_VSCode_BreakpointSttings_ScreenReader:Screen reader does not annouce the checked unchecked status of the checkbox

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#76694]
**Environment Details:**
VSCode Version : 1.36.0
OS Version : Win10
Additional Details:
MAS Violated : MAS 1.3.1

**Repro Steps:**
1) Open Visual studio code editor  
2) Press ctrl+shift+D
3) Navigate to the check Box under Breakdown submenu bar at LHS .
4) Turn the screen reader and observe the screenreader when user check uncheck the checkbox.

**Actual:**
Screen reader does not  announcement the state of the check box

**Expected:**
Screen reader should announce the current state of the this check box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader will face difficulty to access the this controls. 

**MAS Reference:**
MAS 1.3.1 -https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment For Reference:** 
[A11y_VSCode_Breakpoints settings_ScreenReader_DoesNotAnnounceStateOfCheckbox.pptx](https://github.com/microsoft/vscode/files/3362126/A11y_VSCode_Breakpoints.settings_ScreenReader_DoesNotAnnounceStateOfCheckbox.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/52390
